### DISCUSSION TITLE
Competing Structure

### DISCUSSION THREAD

**Discussion by belsrc** [gohypergiant/standard-toolkit#85] [General]
Currently we have to methods (?) when creating code files and test files.

- `foo/index.ts` & `foo/index.test.ts`
- `foo/index.ts` & `foo/foo.test.ts`

Would like to get consensus moving forward.

Personally. to kick it off, I like to use the later. The naming of `index` is kind of forced on us and not something that I particularly care for when naming files. So I usually use the "proper" name on the test files as it has no effect on them one way or the other.

**Comment by brandonjpierce** [comment]
Based on discussing in TSC meeting on 11/12/24 we are going with the rule of simply naming your test file based on the file it is testing against. E.g. `index.ts` and `index.test.ts`

**Comment by belsrc** [comment]
See Issue #93
